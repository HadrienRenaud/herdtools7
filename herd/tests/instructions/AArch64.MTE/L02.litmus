AArch64 L02
Variant=mixed,mte,sync
{
int x[8]={1,2,3,4,5,6,7,8};
0:X1=x;
0:X2=x:red;
}
 P0              ;
 STZ2G  X2,[X1]  ;
 MOV W0,#9       ;
 STR W0,[X2]     ;
 STR W0,[X2,#28] ; 
forall x={9,0,0,0,0,0,0,9}

