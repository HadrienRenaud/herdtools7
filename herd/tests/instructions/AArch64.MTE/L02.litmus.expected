Test L02 Required
States 1
x={9,0,0,0,0,0,0,9};
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (x={9,0,0,0,0,0,0,9})
Observation L02 Always 1 0
Hash=ac1c69c0bcf4aad96545b4f61a6e329b

